src/measure_theory/lp_space.lean

```diff
@@ -158,6 +158,14 @@ begin
   exact ennreal.rpow_lt_top_of_nonneg (le_of_lt hq0_lt) (ne_of_lt hfq),
 end
 
+lemma lintegral_rpow_nnnorm_lt_top_iff_snorm'_lt_top {f : α → F} (hq0_lt : 0 < q) :
+  snorm' f q μ < ∞ ↔ ∫⁻ a, (nnnorm (f a)) ^ q ∂μ < ∞ :=
```

Comment:
The order of the iff in the name and in the property proved are reversed.

Reply:
Thanks!